src/app/progress/Objective.tsx

```diff
@@ -46,10 +45,7 @@ export default function Objective({
   const progressDescription =
     // D1 display description
     (objectiveDef as any).displayDescription ||
-    (!suppressObjectiveDescription && objectiveDef.progressDescription) ||
-    complete
-      ? t('Objectives.Complete')
-      : t('Objectives.Incomplete');
+    (!suppressObjectiveDescription && objectiveDef.progressDescription);
```

Comment:
I think the parens were off before - you want to be able to fall back to Complete/Incomplete only if the progressDescription is empty or suppressObjectiveDescription is true.